instrumentation/opentelemetry-instrumentation-asyncio/src/opentelemetry/instrumentation/asyncio/environment_variables.py

```diff
@@ -0,0 +1,34 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Enter the names of the coroutines to be traced through the environment variable below, separated by commas.
+"""
+OTEL_PYTHON_ASYNCIO_COROUTINE_NAMES_TO_TRACE = (
```

Comment:
@ocelotl are these environment variable names OK? I remember there being some specific scheme for naming them

Reply:
@ocelotl Is it okay to resolve without a specific naming convention?

Reply:
@aabmass what scheme are you referring to?